test/instrumentation/modules/graphql.js

```diff
@@ -62,7 +62,7 @@ test('graphql.graphql - transaction ended', function (t) {
     var trans = data.transactions[0]
 
     t.equal(trans.name, 'foo')
-    t.equal(trans.type, 'custom')
+    t.equal(trans.type, 'graphql')
```

Comment:
Looks like you made these test changes in the wrong file. It's `express-graphql`, not `graphql`.